SlickEngagement_Plugin.php

```diff
@@ -103,6 +103,9 @@ public function addActionsAndFilters()
         // Ensure pages can be configured with categories and tags
         add_action('init', array(&$this, 'add_taxonomies_to_pages'));
 
+        // Register cron hook for processing.
+        add_filter( 'slickstream_fetch_boot_data', array( &$this, 'process_fetch_boot_data' ), 10, 3 );
```

Comment:
I know behind the scene add_action and add_filter is the same.
I wonder if `add_action` is suit more here?